### PR TITLE
Fruits in mixer

### PR DESCRIPTION
**Pull request by JustD4nTe**
User can add fruits via button into mixer, There can be only max 3 fruits. Adding is looped.

### CODE REVIEW

File: components/Mixer.jsx [hunk 1]
```diff
@@ -11,6 +11,39 @@ export default function Mixer() {
         width={240}
         height={520}
       />
+      <div className={styles.insideMixer}>
+        <div className={`${styles.thirdFruit} ${styles.fruitInMixer}`}>
+          {props.fruitsInMixer[2] != null && (
+            <Image
+              src={`/images/fruits/${props.fruitsInMixer[2]}.svg`}
+              alt=""
```
Comment (upsu90) [inline components/Mixer.jsx]: Can you add fruit name in alt?

**Review by upsu90 — CHANGES_REQUESTED** [review]

File: components/Mixer.jsx (hunk 1 above)
Comment (JustD4nTe) [reply components/Mixer.jsx]: Ok

Comment (JustD4nTe) [reply components/Mixer.jsx]: But, why didn't  you add alternative names in FruitIngredients components?

Comment (upsu90) [reply components/Mixer.jsx]: Next to the fruit image we have fruit name so it's unnecessary

**Review by upsu90 — APPROVED** [review]
great! <3

**Pull request merged**